docs/application/dotnet/guides/app-management/message-port.md

```diff
@@ -104,31 +102,31 @@ To send a message from one application (`LocPortApp.Tizen`) to another (`RmtPort
 
     ```
     {
-        _myPort.MessageReceived += MessageReceived_Callback;
-        _myPort.Listen();
+        _rmtPort.MessageReceived += MessageReceived_Callback;
+        _rmtPort.Listen();
     }
 
     private void MessageReceived_Callback(object sender, MessageReceivedEventArgs e)
     {
-        Log.Debug(TAG, "@@@@@@@ Message Received");
-        Log.Debug(TAG, "@@@@@@@ App ID: " + e.Remote.AppId);
-        Log.Debug(TAG, "@@@@@@@ PortName: " + e.Remote.PortName);
-        Log.Debug(TAG, "@@@@@@@ Trusted: " + e.Remote.Trusted);
-        Log.Debug(TAG, "@@@@@@@ message: " + e.Message.GetItem <string> ("message"));
-
-        _remotePort.Send(_confirmMsg, "Tizen.Applications.Tests", "LocalPort");
+        Log.Debug(TAG, "Message Received");
+        Log.Debug(TAG, "App ID: " + e.Remote.AppId);
+        Log.Debug(TAG, "PortName: " + e.Remote.PortName);
+        Log.Debug(TAG, "Trusted: " + e.Remote.Trusted);
+        Log.Debug(TAG, "message: " + e.Message.GetItem <string> ("message"));
     }
     ```
 
-3.  In the sending application, send the message with the `Send()` method of the `Tizen.Applications.Messages.MessagePort` class, providing the message to be sent as an instance of the [Tizen.Applications.Bundle](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Bundle.html) class:
+3.  In the sending application, to send the message should register a local port, call the `Listen()` method of the `Tizen.Applications.Messages.MessagePort` class. And send the message with the `Send()` method of the `Tizen.Applications.Messages.MessagePort` class, providing the message to be sent as an instance of the [Tizen.Applications.Bundle](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Bundle.html) class:
```

Comment:
Can be rephrased as below:

In the sending application, to send the message, follow the steps:

	a.  You must register to a local port.
	
	b.  Call the `Listen()` method `Tizen.Applications.Messages.MessagePort` class.
	
	c.  Use `Send()` method of the `Tizen.Applications.Messages.MessagePort` class, to send the message.
	
	d.  Provide the message to be sent as an instance of the [Tizen.Applications.Bundle](https://developer.tizen.org/dev-guide/csapi/api/Tizen.Applications.Bundle.html) class as shown in the below code: